yaptide/batch/batch_methods.py

```diff
@@ -327,3 +327,27 @@ def delete_job(simulation: BatchSimulationModel, user: KeycloakUserModel,
         return {"message": "Job cancelation failed"}, 500
 
     return {"message": "Job canceled"}, 200
+
+
+def fetch_results_without_stopping(simulation: BatchSimulationModel, user: KeycloakUserModel,
+                                   cluster: ClusterModel) -> tuple[dict, int]:  # skipcq: PYL-W0613
+    """Function that stops simulation without cancelling collection job"""
+    array_id = simulation.array_id
+    tasks = fetch_celery_tasks_by_sim_id(sim_id=simulation.id)
```

Comment:
why do you fetch celery tasks in batch methods ? If I remember correctly batch methods were supposed to run on Ares

Reply:
yes, there is used bad function, i will change it to this one dedicated to batch simulation